src/Uno.UI/UI/Xaml/Controls/BorderLayerRenderer.Android.cs

```diff
@@ -88,7 +88,7 @@ Thickness padding
 					}
 				}
 
-				view.Invalidate();
+				view.RequestLayout();
```

Comment:
No, this is a big deoptimization. In general we don't want to re-layout unless we know we have to, and we can enter this code path for changes that don't affect the measured size of the view (eg Background colour changes).

Try calling RequestLayout() from OnPaddingChanged instead.

Reply:
I moved the logic here just setting a bool in OnPaddingChanged. Could be useful if we need to set it in OnBorderThicknessChanged and OnCornerRadiusChanged